docs/README.md

```diff
@@ -68,14 +68,61 @@ macOS (OS X) development environment.
 # User documentation
 
 * [Getting started (How-To with examples)](usage.md)
+
+## C/C++ Analysis
 * [Analyzer User guide](analyzer/user_guide.md)
-* [Webserver User guide](web/user_guide.md)
-* [Requiring credentials to view analysis results](web/authentication.md)
-* [Overview about connecting multiple analysis run databases](web/products.md)
-* [Permission management](web/permissions.md)
-* [Usage of PostgreSQL database](web/postgresql_setup.md)
-* [How to deal with false positives](analyzer/false_positives.md)
-* [Store Cppcheck analyzer reports](/tools/report-converter/README.md#cppcheck)
+* [Avoiding or suppressing false positives](analyzer/false_positives.md)
+* [Checker and Static Analyzer configuration](analyzer/checker_and_analyzer_configuration.md)
+* [Checker documentation](web/checker_docs.md)
+* [GCC incompatibilities](analyzer/gcc_incompatibilities.md)
```

Comment:
What do you think about a link to the source code comments section here?

Reply:
OK, I added it. And at the same time i moved the source code comments section to the analyzer's user guide. I think it much better fits there, as we are talking about source code specific instruction.s